### ISSUE TITLE
Give your feedback about Silex please

### ISSUE THREAD

**Issue by lexoyo** [silexlabs/Silex#1141]
Here are places where your feedback is very welcome:

* [AlternativeTo](https://alternativeto.net/software/silex-live-web-creation/) 
* [ProductHunt](https://www.producthunt.com/posts/silex/reviews)

**Comment by rayvaughn57** [comment]
Really horrible out of the box Linux Installation. Avoid at all cost.. Docs are the usually hard to follow non descriptive paragraphs .. "..this is how to do it.." ,, and no code preview for the generated pages .. in short don't waste your time

**Comment by lexoyo** [comment]
Thank you for your feedback

You are talking about silex in general and the desktop version in particular right? 

What documentation are you referring to?

**Comment by rayvaughn57** [comment]
My Gripes...

It just doesn't spell out  the why or how to's ...and those environment
variables  why are they  needed ..?? or where that have been set,

outta of the box whether it's the online web version or installed you
can't see any source code or download an example to see if it's worth
buying..

I saw Bootstrap Studio ran a couple of demo's within 5 minutes I had
purchased and never looked back.. 

It's all elctron based  I presume so a Linux installation should be a
snap,,

 Horribly broken installation and guide  and the blogs and FAQ just
mislead those poor developers, really leaving them in the dark and
stranded

**Comment by rayvaughn57** [comment]
Here another problem when attempting to use silexlabs site builder

Save website
Error: I did not manage to save the file. 
Error: Unknown connector: undefined
    at Unifile.callMethod (/usr/local/lib/node_modules/silex-website-builder/node_modules/unifile/lib/index.js:333:41)
    at Unifile.batch (/usr/local/lib/node_modules/silex-website-builder/node_modules/unifile/lib/index.js:324:15)
    at /usr/local/lib/node_modules/silex-website-builder/dist/server/server/router/WebsiteRouter.js:184:21
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/layer.js:95:5)
    at /usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:281:22
    at param (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:354:14)
    at param (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:365:14)
    at param (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:365:14)
    at Function.process_params (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:410:3)
    at next (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:174:3)
    at router (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/usr/local/lib/node_modules/silex-website-builder/node_modules/express/lib/router/layer.js:95:5

**Comment by lexoyo** [comment]
Ok I see
Thx for the answer

**Comment by lexoyo** [comment]
Thank you for your contribution to the Silex community! These engagement posts, showcases, and calls for help played an important role in growing the project.

We're closing older community posts as part of a tracker cleanup so contributors can focus on current v3 work. The community is now on https://community.silex.me — come say hi!
